src/server.h

```diff
@@ -1236,20 +1236,22 @@ struct sharedObjectsStruct {
 };
 
 /* ZSETs use a specialized version of Skiplists */
+/* ZSETs 使用了一种特殊的跳表 */
 typedef struct zskiplistNode {
-    sds ele;
-    double score;
-    struct zskiplistNode *backward;
+    sds ele; /* 存储字符串类型的数据 */
+    double score; /* 排序时使用的分值 */
+    struct zskiplistNode *backward; /* 指向当前节点底层的前一个节点 */
     struct zskiplistLevel {
-        struct zskiplistNode *forward;
-        unsigned long span;
-    } level[];
+        struct zskiplistNode *forward; /* 指向本层的下一节点 */
+        unsigned long span; /* forward 指向的节点与本节点之间的元素个数 */
```

Comment:
```suggestion
        unsigned long span; /* forward 指向的节点与本节点之间的跨度 */
```
应该不能说是元素个数，因为第一个节点和第二个节点的 span 是1，然而它们之间没有元素，应该理解为跨度或者距离。

Reply:
好的，谢谢毛神建议